### PR TITLE
Final predictions

### PR DESCRIPTION
**Pull request by fn2189**
I have change  some of jared functions to make the final prediction but i did it so that all the previous code would work.

### CODE REVIEW

File: exploratory-notebooks/modules/fancy_time_series.py [hunk 1]
```diff
@@ -58,52 +58,59 @@ def get_4_weeks_ago(y, m, d):
...
-        for fld in [
-            'COMPLAINT_YEAR',
-            'COMPLAINT_MONTH',
-            'COMPLAINT_DAY',
-        ]:
-            pred[fld] = bucket[fld]
-        preds.append(pred)
+        try:
+            comparison_fullyear = all_all[
+                (all_all['DECIMAL_DATE'] >= pd.Timestamp(get_52_weeks_ago(
```
Comment (oracleofnj) [inline exploratory-notebooks/modules/fancy_time_series.py:63]: On line 63 you're comparing all_all['DECIMAL_DATE'] to pd.Timestamp(something else)

However, all_all['DECIMAL_DATE'] got created a few lines above (line 57), from the function get_np_dates().

get_np_dates() returns a numpy-format datetime, so you're comparing the thing on the left (not converted with pd.Timestamp) to the thing on the right (which was converted with pd.Timestamp).

**Review by oracleofnj — CHANGES_REQUESTED** [review]
Please see the lines I flagged.

File: exploratory-notebooks/modules/fancy_time_series.py (hunk 1 above)
Comment (fn2189) [reply exploratory-notebooks/modules/fancy_time_series.py:63]: With my version of pandas, once it is stored in a pandas dataframe it is converted  to a timestamp

Comment (fn2189) [reply exploratory-notebooks/modules/fancy_time_series.py:63]: I verified the type before doing this conversion

**Pull request closed without merging**